Lib/_pyrepl/_minimal_curses.py

```diff
@@ -0,0 +1,68 @@
+"""Minimal '_curses' module, the low-level interface for curses module
+which is not meant to be used directly.
+
+Based on ctypes.  It's too incomplete to be really called '_curses', so
+to use it, you have to import it and stick it in sys.modules['_curses']
+manually.
+
+Note that there is also a built-in module _minimal_curses which will
+hide this one if compiled in.
```

Comment:
Is it part of this PR? If not, do you consider adding it or not?

Reply:
Yes, sounds like this comment was carried over from pypy but doesn't apply here.